usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -637,28 +637,41 @@ def fetch_account_details_award(award_id: int) -> dict:
 def fetch_total_outlays(award_id: int) -> dict:
     sql = """
     with date_signed_outlay_amounts (award_id, last_period_total_outlay) as (
-        SELECT faba. award_id, COALESCE(sum(COALESCE(faba.gross_outlay_amount_by_award_cpe,0)
-            + COALESCE(faba.ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe, 0)
-            + COALESCE(faba.ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe, 0)), 0) as last_period_total_outlay
+        SELECT
+            faba.award_id,
+            COALESCE(
+                sum(COALESCE(faba.gross_outlay_amount_by_award_cpe,0)
+                + COALESCE(faba.ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe, 0)
+                + COALESCE(faba.ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe, 0)), 0)
+            as last_period_total_outlay
         FROM
             financial_accounts_by_awards faba
         INNER JOIN submission_attributes sa
             ON faba.submission_id = sa.submission_id
-        INNER JOIN vw_awards a
-            ON faba.award_id = a.id
-            AND a.date_signed >= '2019-10-01'
-        INNER JOIN vw_transaction_normalized tn ON tn.id = a.earliest_transaction_id
-        WHERE sa.is_final_balances_for_fy AND sa.reporting_fiscal_year = tn.fiscal_year AND {award_id_sql}
+        WHERE
```

Comment:
`sa.reporting_fiscal_year = tn.fiscal_year`

Why was this removed?

Reply:
I'd imagine we'd still want it because fiscal year is a query parameter provided by the user